packages/test/local-server-tests/src/test/stagingMode.spec.ts

```diff
@@ -562,25 +552,30 @@ describe("Staging Mode", () => {
 
 			assertConsistent(clients, "States should match after save");
 			assert.equal(
-				reSubmitSquashedLog.length,
+				reSubmitSquashedSpy.callCount,
 				squash === true ? 1 : 0,
 				"Squashed resubmit should be called iff squash = true.",
 			);
-			assert(
-				JSON.stringify((squash ? reSubmitSquashedLog : reSubmitLog)[0][0]).includes(
-					"branch-only",
-				),
-				"Squashed op should contain the edit prefix.",
-			);
-			if (squash !== true) {
+			if (squash === true) {
+				assert(
+					JSON.stringify(reSubmitSquashedSpy.args[0][0]).includes("branch-only"),
+					"Squashed op should contain the edit prefix.",
+				);
+			} else {
 				assert.equal(
-					reSubmitLog.length,
-					// 2 resubmits when disconnected happens because there is one resubmit upon exiting staging mode (to clear staging flags),
-					// then another when we eventually reconnect.
+					reSubmitCoreSpy.callCount,
```

Comment:
```suggestion
					reSubmitCoreSpy.callCount,
					// 2 resubmits when disconnected happens because there is one resubmit upon exiting staging mode (to clear staging flags),
					// then another when we eventually reconnect.
```